components/AboutTitleBar.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div class="AboutTitleBar">
+    <div class="Background-Wrapper">
+      <div class="header-container">
+        <h2>
+          <span class="headertitle">
+            About the Bay Area Pandemic Dashboard
+          </span>
```

Comment:
Minor comment. The span is used to group inline items. Here the h2 is already grouping the title. If you had certain words or icons that need different styling, then the span would make sense so that you can target them specifically.

Reply:
resolved on latest commit